docs/pages/development.js

```diff
@@ -243,6 +243,63 @@ yarn cypress open
 ~~~"
                   />
                 </li>
+                <li>
+                  <Text>
+                    Run{' '}
+                    <Link href="https://github.com/meinaart/cypress-plugin-snapshots" inline>
+                      <Text weight="bold">Cypress snapshot tests</Text>
+                    </Link>
+                    . If any component changes are expected to visually modify your component, you
+                    must update the snapshot tests. In order to run the tests, you must start the
+                    documentation server in one terminal and the Cypress testing interface in
+                    another.
+                  </Text>
+                  <Markdown
+                    text="
+~~~bash
+yarn start
+~~~"
+                  />
+                  <Markdown
+                    text="
+~~~bash
+yarn cypress open
+~~~"
+                  />
+                  <Flex direction="column">
```

Comment:
Could you try placing these side by side, and letting them wrap on small screens? Might help avoid large white space on wide screens

Reply:
DONE